pkgs/development/libraries/libuecc/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchgit, pkgs }:
+
+stdenv.mkDerivation rec {
+  version = "v5";
+  name = "libuecc-${version}";
+
+  src = fetchgit {
```

Comment:
Also a note that versions cannot start with a letter so in a case like this `v` would be moved from the version string to `rev = "refs/tags/v${version}";` otherwise the package name becomes `libuecc-v5`